content/blog/2020/06/2020-07-07-custom-distribution-service.adoc

```diff
@@ -0,0 +1,73 @@
+
+
+=== Quick review
+
+[width="25%",cols="20,^4"]
+|=======
+|Pull Requests Opened |38
+|Github Issues completed |36
+|=======
+
+=== Details
+
+I have written separate blog posts for every week in GSoC and the intricate details for each of them can be found at their respective blog pages. I am including a summary for every phase supported with the respective links. 
+
+==== Community Bonding
+
+This year gsoc had a longer community bonding than any of the previous editions due to the Coronavirus pandemic and therfore this gave me a lot of time to explore, so I spent it by building a prototype for my project. I realised some of the blockages I might face early on, and therfore it gave me more clarity in terms of how I can proceed. I also spent this time preparing a design document which you can find link:https://docs.google.com/document/d/1-ujWVJ2a5VYkUF6UA7m4bEpSDxmb3mJZhCbmoKO716U/edit[here].
+
+link:https://sladyn98.netlify.app/blog/community_bonding/[Community Bonding Blog]
+
+==== Week 1 
+
+In week one, I spent time getting used to the tech stack I would be using, I was pretty familiar with Spring-Boot but react was something I was going to be using for the first time, so I spent time studying more about it. I also got the project page ready, the issues I was going to tackle and the milestones that I had to achieve before the evaluation. I also spent a bit of time setting up the home page and a bit of front-end components.
+
+link:https://sladyn98.netlify.app/blog/gsoc_week1/[Week 1 Blog]
+
+==== Week 2
+
+Once we were done with the initial setup, it was time to work on the core of the project.
+In the second week, I worked on generating the package configuration and the plugin list dummy display page setup.
+I also ran into issues with the Jenkinsfile so the majority of time was spent fixing it.
+Finally I managed to get around those problems.
+You can read more about it in the link:https://sladyn98.netlify.app/blog/gsoc_week2/[Week 2 Blog post].
+
+link:https://sladyn98.netlify.app/blog/gsoc_week2/[Week 2 Blog]
+
+==== Week 3
+
+The last week was spent cleaning up most of the code and getting the remaining milestones in. This was probably the hardest part of phase 1 because it involved connecting the front and back end of the project.You can read more about it here.
+
+link:https://sladyn98.netlify.app/blog/gsoc_week3/[Week 3]
+
+
+=== Getting the Code
+
+The Custom Distribution Service was created from scratch during GSoC and can be found link:https://github.com/jenkinsci/custom-distribution-service[here on Github].
+
+=== Other links
+
+https://docs.google.com/document/d/1C7VQJ92Yhr0KRDcNVHYxn4ri7OL9IGZmgxY6UFON6-g/edit?usp=sharing[GSoC Proposal] +
+https://docs.google.com/document/d/1-ujWVJ2a5VYkUF6UA7m4bEpSDxmb3mJZhCbmoKO716U/edit?usp=sharing[Design Document] +
+https://docs.google.com/document/d/1DSCH-3wh6uV9Rm_j8PcBzq2lvQPhZ31AIwmWkEaLxvc/edit?usp=sharing[Daily Notes] +
+
```

Comment:
Please explain how to reach out to you and what are the next steps. The goal is to facilitate contributions-feedback

Reply:
Yes